src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -413,6 +471,44 @@ public void setValues(JSONObject formData) {
         replicationConfig = ReplicationConfig.createReplicationConfigFromJSON(formData);
     }
 
+    private void initializeGerritCommands() {
+        List<String> stringLabels = new ArrayList<String>(Arrays.asList("verified", "code-review"));
```

Comment:
Slight efficiency if this was a `Set<String>` since `verified` and `code-review` could be added twice, and this list is mostly used for lookup, so a `HashSet` would be quicker at those. But ordering is not preserved if that is important?.